quarkus/admin/src/testFixtures/java/org/apache/polaris/admintool/PostgresTestResourceLifecycleManager.java

```diff
@@ -63,7 +63,10 @@ public Map<String, String> start() {
     context.containerNetworkId().ifPresent(postgres::withNetworkMode);
     postgres.start();
     return Map.of(
-        "polaris.persistence.eclipselink.configuration-file", createPersistenceXml().toString());
+        "polaris.persistence.type",
+        "eclipse-link",
```

Comment:
Is this because all tests currently run against the EclipseLink implementation?

Reply:
If we deprecate EclipseLink, why not purge it from tests too?

Reply:
Let add a separate test profile for EclipseLink then and run maybe a sub-set of tests. How about the bootstrap test in the admin tool and the REST API test in the quarkus server?

Reply:
Note: removing this change actually makes all tests use JDBC, as far as I understand :thinking:

Reply:
It might be best to keep doc changes in this PR and make `application.properties` and test changes in another PR.

Reply:
> Let's add a separate test profile for EclipseLink then and run maybe a sub-set of tests. How about the bootstrap test in the admin tool and the REST API test in the quarkus server?

This profile is just used for eclipse link, the jdbc profile and lifecycle manager is is present is quarkus/test-commons. The reason why i kept both together is i we are deprecating eclipse link via doc and not via code, so hence updating default is in the same PR. Please Let me know your thoughts considering above.

Reply:
ok. thx for the clarification!